src/plugins/flexibleLayout/components/flexibleLayout.vue

```diff
@@ -168,26 +168,30 @@ export default {
         },
         rowsLayout() {
             return this.domainObject.configuration.rowsLayout;
+        },
+        allContainersAreEmpty() {
+            return !this.containers.filter(container => container.frames.length).length;
```

Comment:
may be use `this.containers.every(container => container.frames.length ===0)`

Reply:
I agree, this was old code I was moving around, ill make the change